### ISSUE TITLE
New crate release

### ISSUE THREAD

**Issue by huitseeker** [privacy-scaling-explorations/halo2curves#96]
The repo has recently landed awesome PRs such as #83, as well as PRs such as #86 and #88 that change core data formats. Once #95 lands, it would be great to have a new release of the halo2curves crate, for consumption in projects such as Nova.

**Comment by mratsim** [comment]
Note, for #95, at the current state it will be exposing `jacobi`, `legendre` and the `Legendre` trait.

I don't think I've heard any direction regarding where to go with the API

> 1. AFAIK it was because exponentiation by ``(p-1)/2`` needed to abstract away the usage of curve-specific modulus constants.
In that case, the trait can be deleted, it should be recent enough that breakage is minimal if any (cc @huitseeker).
> 2. Alternatively, the `legendre` procedure can call `jacobi` instead and the `Legendre` trait can be preserved. Though I'm curious what's the use-case of a trait instead of a function in that case.

I'm personally in-favor of removing the Legendre trait.

Another change of note is #86, we need to remove the MSM/FFT impl from halo2 repo when upgrading it to avoid ambiguous calls.

**Comment by CPerezz** [comment]
Would be nice to have some sort of solution for https://github.com/zkcrypto/group/pull/48 prior to releasing. But we can release a patch on the meantime if needed. 

Agree then we need to address the comments of @mratsim.

**Comment by mratsim** [comment]
closed by #114
